charts/nexus-repository-manager/tests/ingress_test.yaml

```diff
@@ -97,7 +98,22 @@ tests:
         equal:
           path: metadata.name
           value: RELEASE-NAME-nexus-repository-manager
-
+      - documentIndex: 0
+        equal:
+          path: spec
+          value:
+            ingressClassName: nginx
+            rules:
+              - host: repo.demo
+                http:
+                  paths:
+                    - path: /
+                      pathType: Prefix
+                      backend:
+                        service:
+                          name: RELEASE-NAME-nexus-repository-manager
+                          port:
+                            number: 8081
```

Comment:
show normal existence of ingressClassName for repo url to contrast for when it's not specified in a later test.